newIDE/app/src/UI/Dialog/index.js

```diff
@@ -102,8 +110,25 @@ export default (props: Props) => {
     <ResponsiveWindowMeasurer>
       {size => (
         <Dialog
+          onApply={onApply}
```

Comment:
This line is doing nothing :) `Dialog` here is a material-ui Dialog, which [has no `onApply` prop to be passed](https://material-ui.com/api/dialog/). 

onApply here is a prop **passed by** the component using the Dialog (for example, `NewInstructionEditorDialog.js`)

Reply:
Yeah i've seen that, a <Dialog> from material-ui in another custom <Dialog> 🤨
I've rewrite import with import as for a better understanding.

What is for `onApply` props for a `<Dialog>` like in [NewObjectDialog.js](https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/AssetStore/NewObjectDialog.js)
I think there is no reason to add it here, but in this case i have to check in  `<Dialog>` if the props exist before call it.

Reply:
> I've rewrite import with import as for a better understanding.

Ok 👍

> What is for onApply props for a `<Dialog>` like in NewObjectDialog.js
> I think there is no reason to add it here, but in this case i have to check in `<Dialog>` if the props exist before call it.

Correct. There is no reason to add it (the dialog can just be closed or a choice done), so a backdrop click (or Escape) should result in calling onRequestClose (because onApply is undefined).